### PR TITLE
Feature/therapist show page

### PR DESCRIPTION
**Pull request by ManiekCR**
Capture d’écran 2019-08-27 à 22 32 55

First version of the therapists show page done, I've made a main container which is flex, with 3 items (1- avatar and content below it, 2- biocard + reviewcard, 3- bookingcard (form, which is sticky)).

I've added a bit of ruby code (for specialties, first name, last name and years of experience).

For the rest I took from our sandbox in order to see if the display is good :)

### CODE REVIEW

**Comment by ManiekCR** [comment]
Oh and don't worry about the doctor name, that's what I created in my local database :D

File: app/assets/stylesheets/therapists/_show.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.show-container {
```
Comment (sbalsom) [inline app/assets/stylesheets/therapists/_show.scss]: will you make your classes specific to your page, aka 'therapist-show-container'

File: app/assets/stylesheets/therapists/_show.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  display: flex;
+  padding: 40px 20px 20px 10px;
+  justify-content: space-around;
+}
+
+.show-first-flex-item div {
```
Comment (sbalsom) [inline app/assets/stylesheets/therapists/_show.scss]: will you please nest your classes as in SCSS style aka ;

.show-therapist-container {
.show-first-flex-item {
}
}

File: app/assets/stylesheets/therapists/_show.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  text-align: center;
+  padding-top: 20px;
+}
+
+.show-form {
```
Comment (sbalsom) [inline app/assets/stylesheets/therapists/_show.scss]: also please nest these

File: app/assets/stylesheets/components/_bookingCard.scss [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
   position: relative;
-  margin: 20px auto;
   width: 380px;
   padding: 45px 50px 70px 50px;
   border-radius: 1px;
   border: 1px solid $border;
-  height:100%;
 }
 
 .booking-title {
```
Comment (sbalsom) [inline app/assets/stylesheets/components/_bookingCard.scss]: also please nest these

**Pull request merged**